src/assets/wise5/components/common/feedbackRule/TermEvaluator/IsSubmitNumberEvaluator.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { CRaterResponse } from '../../cRater/CRaterResponse';
+import { IsSubmitNumberEvaluator } from './IsSubmitNumberEvaluator';
+
+describe('IsSubmitNumberEvaluator', () => {
+  describe('evaluate()', () => {
+    it("should return true iff response's submitCounter is the expected submit number", () => {
+      const response_2nd_submit = createResponseWithSubmitNumber(2);
+      const response_3rd_submit = createResponseWithSubmitNumber(3);
+      const evaluator_2 = new IsSubmitNumberEvaluator('isSubmitNumber(2)');
+      expect(evaluator_2.evaluate(response_2nd_submit)).toBeTrue();
+      expect(evaluator_2.evaluate(response_3rd_submit)).toBeFalse();
+
+      const evaluator_10 = new IsSubmitNumberEvaluator('isSubmitNumber(10)');
+      expect(evaluator_10.evaluate(response_2nd_submit)).toBeFalse();
+      const response_10th_submit = createResponseWithSubmitNumber(10);
+      expect(evaluator_10.evaluate(response_10th_submit)).toBeTrue();
+    });
```

Comment:
Should this be two tests?

Reply:
Do you mean separate into
```"should return true iff response's submitCounter is the expected submit number (single-digit)"```
and
```"should return true iff response's submitCounter is the expected submit number (double-digit)"```
?

I thought about that, but wasn't sure, so I put it into one and separated with a newline. In both cases, it's testing the same block of code in ```evaluate()```- the code doesn't handle single and double digit inputs differently, and they return the same output. I think it makes sense to have two tests if the input tests different parts of the code, and you expect different behavior based on the input.

What do you think?

Reply:
You're right. It's not necessary.